README.md

```diff
@@ -79,6 +79,19 @@ It starts etcd and simulator-server locally.
 
 To run the frontend, please see [README.md](./web/README.md) on ./web dir.
 
+## Existing cluster importing
+The simulator can import existing clusters.
+This allows for batch inclusion of resources from external clusters.
+It is enabled by an `EXTERNAL_IMPORT_ENABLED` environment variables is `1`.
+
+The following environment variables configure this function:
+- `EXTERNAL_IMPORT_ENABLED`:If it is `1`, this importing function is enabled.
+  The scheduler starts importing from external clusters as soon as it starts.
+- `KUBECONFIG`:The cluster from which you are importing is indicated by this environment variable.
+  [Read more](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/#the-kubeconfig-environment-variable)
+- `KUBE_SCHEDULER_CONFIG_PATH`:This is an optional. The simulator can't import the scheduler configuration from existing cluster via `kube-apiserver`.
```

Comment:
I think we don't need to write a doc for `KUBE_SCHEDULER_CONFIG_PATH` here.
We should write it on #138.
cc @pravarag

Reply:
Also, @sanposhiho how about we include `KUBECONFIG` and `EXTERNAL_IMPORT_ENABLED` fields as well in #138 ? We can then refer it all together in the README.md file.

Reply:
👍 

Having the same explanation in two places can lead to forgetting it when updating the doc in the future. So, I think it's better to add doc about KUBECONFIG and EXTERNAL_IMPORT_ENABLED in #138 and just add the link to docs/env-variables.md from here as @pravarag suggested.

maybe like:

```markdown

## [Beta] Existing cluster importing

You can import resources on your cluster to the simulator by using the following environment variables `EXTERNAL_IMPORT_ENABLED` and `KUBECONFIG`.
For more information about those variables, please refer [docs/env-variables.md](docs/env-variables.md).
```

Reply:
yup, thanks! I have replaced all of your suggestions. So, I wait for #138 to be finished and merged.

/hold again